tools/preseal/preseal.c

```diff
@@ -33,6 +33,7 @@
 
 #define DEFAULT_PCR_INDEX 16
 
+#ifndef NO_PRESEAL_FILESYSTEM
```

Comment:
stdio.h, fcntl.h and unistd.h may need to be guarded from inclusion as well if there is no filesystem.